.ci/packaging.groovy

```diff
@@ -218,6 +244,32 @@ def release(){
   }
 }
 
+def triggerE2ETests(String suite, String channel) {
+  echo("Triggering E2E tests for ${env.BEATS_FOLDER}. Test suite: ${suite}. Build result will be sent the ${channel} Slack channel")
```

Comment:
is this common for PR CI?  I'm not sure we want the slack channel pinged, maybe just the individual pr creator, or not at all (which is fine)?  As long as it will be seen in the PR it isn't needed to notify the main slack channel, it'll be too much.

Reply:
Sure. I'll add that PRs won't send to slack in the downstream job (https://github.com/elastic/e2e-testing/blob/master/.ci/Jenkinsfile#L238-L245). So maybe that message is superfluous